apps/alpha-test-app/src/App.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+import { provideI18n } from "sit-onyx";
+import deDE from "sit-onyx/locales/de-DE.json";
+import { useI18n } from "vue-i18n";
+import HelloWorld from "./components/HelloWorld.vue";
+
+const { locale } = useI18n();
+provideI18n({ locale, messages: { "de-DE": deDE } });
```

Comment:
<img width="961" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/67a56b5c-2115-46b5-b3d4-43e2909a500a">
@larsrickert is this error known to you?

Reply:
deDE as OnyxTranslations

-> typecast solves issues.

Should we add this to the docs that this is needed? :/

Reply:
Actually the definition of our `OnyxTranslations` is invalid. Can you change it to in `packages/sit-onyx/src/src/i18n/index.ts`:
```ts
/**
 * The type of the imported `enUS` above is a concrete type so value type of each message
 * is e.g. "{ myKey: 'HelloWorld'}" but it should be "{ myKey: string }".
 * This utility type converts all values to be of type string (more generic)
 * so we can actually use other locales as well.
 */
type GetTypeOfTranslations<T> = T extends object
  ? { [P in keyof T]?: GetTypeOfTranslations<T[P]> }
  : string;

/** Available translations that are used by onyx components. */
export type OnyxTranslations = GetTypeOfTranslations<typeof enUS>;
```

Reply:
will do

Reply:
nice, that's way better 😸